documentation/biblatex-manuscripts-philology.tex

```diff
@@ -139,7 +169,10 @@ \subsection{Special}\label{fields:special}
   catalog = {[prenote1][postnote1]{key1}[prenote2][postnote2]{key2}}
 \end{minted}
 
-\fielditem{content}{literal} description of the manuscript's content, can be used instead of \bibfield{annotation}. If you want to add paragraphs inside it, you must use the \cs{par} command between each paragraph.
+\fielditem{content}{literal} description of the manuscript's content, can be used 
+with or instead of \bibfield{annotation}. It starts a new section of the description 
+output. If you want to add paragraphs inside it, you must use the \cs{par} 
+command between each paragraph.  E.g., ``Covers chapters 1--3 only.''
```

Comment:
I don't understand " It starts a new section of the description  output"

Reply:
I mean like what biblatex calls a "block".  At present, there's a heading "Contents" and another "Remarks," each starting a new section of the layout.  That's what I mean.  Contents and Annotation fields are not run together in a single paragraph.

Reply:
in this case, I will use "bllock" terme. Section can refer to \section, or to \beginnumbering... we should not add a third use!

Reply:
corrected